modules/dnn/test/test_ie_models.cpp

```diff
@@ -227,8 +227,24 @@ static testing::internal::ParamGenerator<String> intelModels()
     return ValuesIn(modelsNames);
 }
 
+static testing::internal::ParamGenerator<Target> dnnDLIETargets()
+{
+    std::vector<Target> targets;
+    targets.push_back(DNN_TARGET_CPU);
+#ifdef HAVE_OPENCL
+    if (cv::ocl::useOpenCL() && ocl::Device::getDefault().isIntel())
+    {
+        targets.push_back(DNN_TARGET_OPENCL);
+        targets.push_back(DNN_TARGET_OPENCL_FP16);
+    }
+#endif
+    //if (checkMyriadTarget())
+    //    targets.push_back(DNN_TARGET_MYRIAD);
```

Comment:
There is an issue with multiple applications which use one stick. OpenCV loads myriad plugin once so test for OpenVINO models cannot load a model after OpenCV. Let's keep it as TODO for the future changes.